src/styles/_catalog.scss

```diff
@@ -1,22 +1,24 @@
+@import "./utils/variables";
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    margin-bottom: $gap;
   }
 
   &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    margin-right: $gap;
```

Comment:
need to remove this property because it adds your border as well in the gap
use `gap` property instead